### DISCUSSION TITLE
Avast One detects thread on self-hosted Plausible script

### DISCUSSION THREAD

**Discussion by wichtounet** [plausible/analytics#3539] [Self-Hosted Support] [answered]
### Past Issues Searched

- [X] I have searched open and closed issues to make sure that the bug has not yet been reported

### Issue is a Bug Report

- [X] This is a bug report and not a feature request, nor asking for self-hosted support

### Using official Plausible Cloud hosting or self-hosting?

Self-hosting

### Describe the bug

In the last few weeks, I got several emails from my readers indicating that Avast detects incidents on my blog. And in all occurrences, this is related to the script.js script from Plausible. I am using self-hosted on https://thepoorswiss.com

The alert is shown regularly any then any call to the subdomain (s.thepoorswiss.com) is then blocked. I don't care about the missing stats, but this will likely scare people away. 

Is there anything I can do? Has anybody seen such issue?

### Expected behavior

The expected behavior would be that the script is not flagged as malicious for my readers. 

### Screenshots

![Photo from Baptiste](https://github.com/plausible/analytics/assets/196746/86374ebf-9e3c-44a1-b9bd-1dfd6a22b3e1)

### Environment

```markdown
- OS: Windows
- Browser: Chrome
- Browser Version: Any, with Avast running
```

**Comment by ruslandoga** [comment]
👋 @wichtounet 

I've moved this thread to discussions since it's unlikely to be an issue with the code.

**Comment by ruslandoga** [comment]
What's under "See details" on your screenshot? Right now it's difficult to tell what might be causing problems.

**Comment by wichtounet** [comment]
<img width="468" alt="Capture" src="https://github.com/plausible/analytics/assets/196746/4acebf94-8a1e-4e34-9ddb-b33ee52c0e36">

Unfortunately, the details are not very useful.

**Comment by povpie** [comment, marked as answer]
I suggest sending a false positive report [here](https://www.avast.com/false-positive-file-form.php#pc)

**Comment by wichtounet** [comment]
They did not provide any detail, but they have cleared the warning. So, it may have nothing to do with Plausible. Thanks for the support 👍

**Reply by kpapamanos** [reply]
@wichtounet Did they answer to your report or they just fixed it in their database without notifying?
I had multiple user reports for several of our sites. I submitted a false positive report today and i am waiting for a response. 🤷🏻‍♂️

**Reply by wichtounet** [reply]
@kpapamanos They told me they fixed it in the database, without any further details, but at least they answered. But it came back after a few days. Now, it seems okay after three or four reports, but I am not sure it will last. The next time it happens, I will seriously consider dropping the analytics.

**Comment by lubomirblazekcz** [comment]
We had the same problem, we have sent a false positive report.
<img width="607" alt="obrazek" src="https://github.com/plausible/analytics/assets/6872956/c4019fd3-3493-4cca-b4bb-68cc336a8c27">

**Comment by LagClear** [comment]
I am receiving this issue along side other friends who self host plausible. I have emailed and sent in a whitelisting request and haven't gotten any response any suggestions?

**Comment by jwoertink** [comment]
My site is now getting this too. I'm currently telling all of my customers to go and report the false positive, but for new users visiting the site, we're seeing people get nervous since it says "Phishing". My guess is it was related to the outbound-link tracking, so we disabled that, but it didn't seem to fix it. Thankfully most our customers are on mobile and aren't affected :sweat_smile:
